laterpay/views/frontend/partials/post/overlay_with_purchase_button.php

```diff
@@ -20,51 +20,73 @@
+                <?php else if ( ! $laterpay['only_time_pass_purchases_allowed'] && $laterpay['revenue_model'] == 'sis' ): ?>
+                    <li class="lp_benefits__list-item lp_benefit--buy-now">
+                        <h3 class="lp_benefit__title"><?php _e( 'Buy Now', 'laterpay' ); ?></h3>
+                        <p class="lp_benefit__text">
+                            <?php _e( 'Buy this post now with LaterPay and <br>pay with a payment method you trust.', 'laterpay' ); ?>
+                        </p>
+                    </li>
+                    <li class="lp_benefits__list-item lp_benefit--use-immediately">
+                        <h3 class="lp_benefit__title"><?php _e( 'Read Immediately', 'laterpay' ); ?></h3>
+                        <p class="lp_benefit__text">
+                            <?php _e( 'Immediately access your purchase. <br>You only buy this post. No subscription, no fees.', 'laterpay' ); ?>
+                        </p>
+                    </li>
+                <?php else: ?>
+                    <li class="lp_benefits__list-item lp_benefit--buy-now">
+                        <h3 class="lp_benefit__title"><?php _e( 'Buy Now', 'laterpay' ); ?></h3>
+                        <p class="lp_benefit__text">
+                            <?php _e( 'Just agree to pay later.<br> No upfront registration and payment.', 'laterpay' ); ?>
+                        </p>
+                    </li>
+                    <li class="lp_benefits__list-item lp_benefit--use-immediately">
+                        <h3 class="lp_benefit__title"><?php _e( 'Read Immediately', 'laterpay' ); ?></h3>
+                        <p class="lp_benefit__text">
+                            <?php _e( 'Get immediate access to your purchase.<br> You are only buying this article, not a subscription.', 'laterpay' ); ?>
+                        </p>
+                    </li>
+                    <li class="lp_benefits__list-item lp_benefit--pay-later">
+                        <h3 class="lp_benefit__title"><?php _e( 'Pay Later', 'laterpay' ); ?></h3>
+                        <p class="lp_benefit__text">
+                            <?php _e( 'Buy with LaterPay until you reach a total of 5 Euro.<br> Only then do you have to register and pay.', 'laterpay' ); ?>
+                        </p>
+                    </li>
+                <?php endif; ?>
+            </ul>
+            <div>
+                <?php if ( $laterpay['only_time_pass_purchases_allowed'] ): ?>
+                    <a href="#lp_js_timePassWidget"
+                        class="lp_purchase-button"
+                        title="<?php echo __( 'View available LaterPay Time Passes', 'laterpay' ); ?>"
+                        data-icon="b"><?php _e( 'Get a Time Pass', 'laterpay' ); ?></a>
+                <?php else: ?>
                     <?php if ( defined( 'DOING_AJAX' ) && DOING_AJAX ): ?>
                         <?php
                             ob_start();
```

Comment:
why do we need this? looks bad in template, if this is really required should be moved to controller or helper